_posts/2017-06-09-Principles-of-object-oriented-design.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+author: domenique
+disqus: false
+header-img: "img/patterns.jpg"
+---
+
+_SOLID_, The famous acronym coined by Micheal Feathers describing the first five principles of object oriented design composed by Robert C. Martin a.k.a Uncle Bob. What most people tend to forget is that __SOLID describes only five of the in total eleven principles of object oriented design__. These principles date all the way back to 1995, yes Uncle Bob is really that old :)
```

Comment:
Voor te starten met "what most people tend to forget" zou ik eerst nog een paar extra zinnen aan SOLID spenderen. De zin komt enerzijds licht agressief over maar springt ook te snel van de hak op de tak. De post gaat over de SOLID principles. Zeggen dat er nog andere zijn in de tweede zin is niet nodig. Ik zou eerst het belang van de principes duidelijk maken. Waarom moeten mensen deze post lezen?